services/api-server/src/simcore_service_api_server/core/application.py

```diff
@@ -5,10 +5,16 @@
 from models_library.basic_types import BootModeEnum
 from packaging.version import Version
 from servicelib.fastapi.profiler_middleware import ProfilerMiddleware
-from servicelib.fastapi.tracing import setup_tracing
+from servicelib.fastapi.tracing import (
+    setup_tracing,
+    setup_tracing_spans_for_simcore_service_functions,
+)
 from servicelib.logging_utils import config_all_loggers
+from simcore_service_api_server import exceptions
+
+setup_tracing_spans_for_simcore_service_functions()
```

Comment:
should this not be called in your import directly? I have the feeling this is very fragile as this will for sure move with some of our tools.

also, this instruments everything? I guess my questions above then remain more than ever.